services/apis/indexd/indexdTasks.js

```diff
@@ -82,27 +86,71 @@ module.exports = {
    * @param {Object} fileNode - Assumed to have a did property
    * @returns {Promise<Gen3Response>}
    */
-  async getFile(file) {
+  async getFile(file, headers = user.mainAcct.accessTokenHeader) {
     // get data from indexd
     return I.sendGetRequest(
       `${indexdProps.endpoints.get}/${file.did}`,
-      user.mainAcct.accessTokenHeader,
+      headers,
     ).then((res) => {
       file.rev = getRevFromResponse(res);
       return res.body;
     });
   },
 
+  /**
+   * Updates indexd data for file
+   * @param {Object} fileNode - Assumed to have a did property
+   * @returns {Promise<Gen3Response>}
+   */
+  async updateFile(file, data, headers = user.mainAcct.indexdAuthHeader) {
```

Comment:
this is backwards compatible in that it will default to sending auth header both times (which should be fine for GET in old indexd). then in new indexd, it needs access token for both, so we just pass in the headers arg in those tests

Reply:
oh i see! let's rename the param `authHeaders` to make it clear then? and change which headers you pass to this function in the tests